components/mpas-ocean/src/Registry.xml

```diff
@@ -655,6 +659,20 @@
 					description="Option of Langmuir enhanced entrainment parameterization"
 					possible_values="NONE, LWF16, LF17, RWHGK16"
 		/>
+		<nml_option name="config_cvmix_kpp_use_active_wave" type="logical" default_value=".false." units="NA"
+					description="Flag for Langmuir enchancement factor using prognostic waves"
+					possible_values=""
+		/>
+	</nml_record>
+	<nml_record name="wave_coupling" mode="forward">
+		<nml_option name="config_use_active_wave" type="logical" default_value=".false." units="NA"
+					description="Flag for using prognostic waves"
```

Comment:
Does this flag turn on coupling with WW3? Could you add more to the description here: 
1. is this required to be on with WW3 coupling in E3SM
2. What happens if you turn this on in stand-alone, or in E3SM without WW3?

It seems like the master switch for WW3 should be in E3SM, not here. But perhaps this just says whether to apply the coupling fields in MPAS-Ocean.

Reply:
@mark-petersen, thanks for bringing this up; I wanted to make sure I got feedback about this. This isn't required for WW3 coupling in E3SM. Right now this flag only controls the MPAS-O package for waves, so that none of the waves related arrays are allocated when WW3 isn't being used.

Reply:
In that case, I would name it `config_allocate_wave_arrays`. But it must also control the use of those arrays within MPAS-Ocean? Or the planned use of them in the future? Otherwise, what would the arrays be for?

Reply:
I forgot to mention that right now this also controls the calculation of the stokes drift profiles as well. Another thing to mention is that I've been using MPAS streams as the primary means of outputting wave variables because 1) the WW3 output is really cumbersome, and 2) it makes it possible to use MPAS-Analysis for waves. So this flag also facilitates that streams output in absence of coupled parameterizations.